plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/ExtDirect.js

```diff
@@ -75,7 +75,11 @@ Ext.define('NX.controller.ExtDirect', {
     if (message) {
       NX.Messages.add({text: message, type: 'warning'});
     }
-    me.logDebug(transaction.action + ':' + transaction.method + " -> " + (message ? 'Failed: ' + message : 'OK'));
+    var logMsg = transaction.action + ':' + transaction.method + " -> " + (message ? 'Failed: ' + message : 'OK');
```

Comment:
adding this as validation errors can be included even when 'OK' is logged to the console